lib/awful/screen.lua

```diff
@@ -201,6 +203,28 @@ function screen.object.get_tiling_area(s)
     }
 end
 
+--- Take a screenshot of the physical screen.
+--
+-- Reading this (read only) property returns a screenshot of the physical
+-- (Xinerama) screen as a cairo surface.
+--
+-- @DOC_screen_content_EXAMPLE@
```

Comment:
there is no example itself in `./tests/examples/awful/screen` as far as i see from this patch